src/Model/Contact/Relation.php

```diff
@@ -482,12 +482,11 @@ static public function getSuggestions(int $uid, int $start = 0, int $limit = 80)
 	 */
 	public static function countFollows(int $cid, array $condition = []): int
 	{
-		$condition = DBA::mergeConditions($condition, [
-			'`id` IN (SELECT `relation-cid` FROM `contact-relation` WHERE `cid` = ? AND `follows`)',
-			$cid,
-		]);
+		$condition = DBA::mergeConditions($condition, ["`cid` = ? and `follows`", $cid]);
+		$sql = "SELECT COUNT(*) AS `total` FROM `contact-relation` INNER JOIN `contact` ON `contact`.`id` = `relation-cid` WHERE " . array_shift($condition);
```

Comment:
How does the `array_shift()` works in relation with the later `fetchFirst()` call?

Reply:
The first array element contains the query, the other elements are the parameters. The `fetchFirst` needs the SQL string as first parameter and then the parameters either as array or as individual parameters.